client_core/src/com/mygdx/game/client_core/network/service/GameConnectService.java

```diff
@@ -21,9 +21,9 @@ public GameConnectService(
     this.playerInfo = playerInfo;
   }
 
-  public void connect() {
+  public void connect(String gameRoomName) {
     log.info("connect request sent");
-    sender.send("connect:" + playerInfo.getUserName() + ":" + playerInfo.getToken() + ":default");
+    sender.send("connect:" + playerInfo.getUserName() + ":" + playerInfo.getToken() + ":" + gameRoomName);
   }
```

Comment:
tutaj można wstawiać parametry do pokoju - na serwerze może być tak, że jeśli pokój jest nieutworzony to wszystkie parametry są potrzebne i obsługiwane, a jak już istnieje to tylko nazwa